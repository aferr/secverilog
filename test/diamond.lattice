(declare-fun D1 () Label)
(declare-fun D2 () Label)
(assert (and (not (leq D1 D2)) (not (leq D2 D1))))