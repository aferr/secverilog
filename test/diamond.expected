Compiling file diamond.v
Verifying file diamond.v
fail
(assert  (not(leq D1  D2)))    ; d2 = d1 @diamond.v:33
(assert  (not(leq D2  D1)))    ; d1 = d2 @diamond.v:38
Total: 2 assertions failed
